src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -707,6 +707,12 @@ private void Dispose(bool disposing)
 
             if (NetEventSource.Log.IsEnabled()) NetEventSource.Info(_state, $"{TraceId()} disposing {disposing}");
 
+            // If we haven't already shutdown gracefully (via a successful CloseAsync call), then force an abortive shutdown.
+            MsQuicApi.Api.ConnectionShutdownDelegate(
+                _state.Handle,
+                QUIC_CONNECTION_SHUTDOWN_FLAGS.SILENT,
+                0);
```

Comment:
It feels like it should be configurable from somewhere, but we might want to at least have it as a constant `DEFAULT_ABORT_CODE` or something like this.

Reply:
This value doesn't actually matter, because of the SILENT flag -- no error is sent to the peer.

In general though, I do agree we should define DEFAULT_ABORT_CODE or something like that.